webapp/channels/src/packages/mattermost-redux/src/actions/channels.ts

```diff
@@ -1426,6 +1430,31 @@ export function getChannelMemberCountsByGroup(channelId: string) {
     });
 }
 
+export function fetchMissingChannels(channelIDs: string[]): ActionFuncAsync<Array<Channel['id']>> {
+    return async (dispatch, getState, {loaders}: any) => {
+        if (!loaders.missingChannelLoader) {
+            loaders.missingChannelLoader = new DelayedDataLoader<Channel['id']>({
+                fetchBatch: (channelIDs) => {
+                    return channelIDs.length ? dispatch(getChannel(channelIDs[0])) : Promise.resolve();
+                },
+                maxBatchSize: maxMissingChannelsBatchSize,
+                wait: missingChannelWait,
+            });
+        }
+
+        const state = getState();
+        const missingChannelIDs = channelIDs.filter((channelId) => !getChannelSelector(state, channelId));
+
+        if (missingChannelIDs.length > 0) {
+            await loaders.missingChannelLoader.queueAndWait(missingChannelIDs);
```

Comment:
Since we're not waiting on the `fetchMissingChannels` call, could you change this to just use `queue` and have the action return something simpler like `data: true`? `queueAndWait` requires some mildly complicated logic to keep track of who's waiting on what, so I'd like to avoid it unless we know we need it